mono/eglib/glib.h

```diff
@@ -399,6 +402,7 @@ gchar  *g_stpcpy             (gchar *dest, const char *src);
 gchar   g_ascii_tolower      (gchar c);
 gchar   g_ascii_toupper      (gchar c);
 gchar  *g_ascii_strdown      (const gchar *str, gssize len);
+void    g_ascii_strdown_no_alloc (char* dst, const char* src, gsize len);
```

Comment:
Is there a reason you're using `char` instead of `gchar` here? Also style nit: should be `char *dst`.

Reply:
There is no point to `gchar`. Prefer `char`.

Reply:
`char* dst` vs. `char *dst` is debatable.
The second tends to be preferred by people who want to highlight C's backwardness.
The first is for people who want it to look sane despite the underlying craziness.
C's declaration style is a failed experiment in coming up with grammars that require right to left parsing instead of the intuitive left to right.

Reply:
Yes, but most of the file (and the runtime as a whole) uses the second option. I personally prefer the first but consistency is what's important here.

Reply:
As far as the `gchar` thing, I was going to say it's for consistency there as well but looking at the header we seem to be all over the place for no good reason. Is there any logic to it? cc @lambdageek

Reply:
Maybe sometime styles will be revisited and a better consistency will be established instead of being stuck with old less favored styles forever. :(

Reply:
Okay I think char is fine, so just the minor stylistic nit with the asterisk